plugins/catalog/src/components/SystemDiagramCard/SystemDiagramCard.tsx

```diff
@@ -128,7 +128,9 @@ function RenderNode(props: DependencyGraphTypes.RenderNodeProps<any>) {
           alignmentBaseline="baseline"
           style={{ fontWeight: 'bold' }}
         >
-          {props.node.name}
+          {props.node.name.length < 20
```

Comment:
Probably nicest to declare this as `const name = ...length > 20: ? props.name` at the top with the other variables to make the render code a bit cleaner.

Reply:
Done! Hoisted it up, and pulled out the `20` int to a constant. Wasn't sure if you had any preferences on where the constants live, I chose to just keep it near to where it's used, but could be pulled up to the top of the module. 

```javascript
  const classes = useStyles();
  const catalogEntityRoute = useRouteRef(entityRouteRef);
  const kind = props.node.kind || 'Component';
  const ref = parseEntityRef(props.node.id);
  const MAX_NAME_LENGTH = 20;
  const truncatedNodeName = props.node.name.length < MAX_NAME_LENGTH ?
    props.node.name :
    `${props.node.name.slice(0, MAX_NAME_LENGTH)}...`
```